tests/securedrop/server_start.pm

```diff
@@ -0,0 +1,49 @@
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+use base "installedtest";
+use strict;
+use testapi;
+use networking;
+use serial_terminal qw(select_root_console);
+
+
+sub run {
+    my ($self) = @_;
+    select_root_console;
+
+    $self->select_gui_console;
+
+    # Update onion address
+    x11_start_program('xterm');
+
+    background_script_run("qvm-run -p sd-dev \"cd securedrop\; sed -i 's|/dev/stdout|/dev/null|g' securedrop/bin/dev-shell && make dev-tor\" </dev/null 2>&1 >/dev/null"); # | sed 's/^/[SD Server] /'"); # grep "journalist interface" so that it does not interfere with needles
```

Comment:
Fine for this PR, but let's move any hacks like this into the SD server repo itself, guarded by an OPENQA environment variable or something.